Tasks/PyPIPublisher/publisher.ts

```diff
@@ -0,0 +1,34 @@
+var tl = require('vsts-task-lib');
+var workingDirectory = tl.getInput('wd', true);
+var serviceEndpointId = tl.getInput('serviceEndpoint', true);
+var os = require('os');
+var shell = require('shelljs');
+var homedir = os.homedir();
+
+//Generic service endpoint
+var auth = tl.getEndpointAuthorization(serviceEndpointId, false);
+var pythonServer = tl.getEndpointUrl(serviceEndpointId, false);
+var username = auth['parameters']['username'];
+var password = auth['parameters']['password'];
+
+//Create .pypirc file
+var text = "[distutils] \nindex-servers =\n    pypi \n[pypi] \nrepository=" + pythonServer + " \nusername=" + username + " \npassword=" + password;
```

Comment:
is writing pasword in this file the only option? Can password be passed to commandline or passed through a file or env variable?